docs/content/getting-started/create-new-project.mdx

```diff
@@ -58,7 +58,7 @@ dagit
 
 The Dagit process automatically uses the file `workspace.yaml` to find your repositories, from which Dagster will load your jobs, schedules, and sensors. To see how you can customize the Dagit process, run `dagit --help`.
 
-3. (Optional) If you want to enable Dagster [Schedules](/concepts/partitions-schedules-sensors/schedules) or [Sensors](/concepts/partitions-schedules-sensors/sensors) for your jobs, start the [Dagster Daemon process](/deployment/dagster-daemon) **in the same folder as your `workspace.yaml` file, but in a different shell or terminal**:
+3. (Optional) If you want to enable Dagster [Schedules](/concepts/partitions-schedules-sensors/schedules) or [Sensors](/concepts/partitions-schedules-sensors/sensors) for your jobs, start the [Dagster Daemon process](/deployment/dagster-daemon) **in the same folder as your `workspace.yaml` file, but in a different shell or terminal**. Make sure `$DAGSTER_HOME` is set to somewhere other than `/tmp`:
```

Comment:
We should really be doing one of three things here:

1. Preventing you from setting `$DAGSTER_HOME` to anything in `/tmp`.
2. Choosing a default and warning if one isn't provided (a la running `dagit`)
3. Altering the warning text to tell you not to use `/tmp`.

I think it's important that we either choose for the user or prompt in the tutorial to set _something_ - otherwise it's a bit jarring to run the command as written in the tutorial and then to immediately encounter an error.